src/ILICheck.Web/GeoJsonHelper.cs

```diff
@@ -0,0 +1,100 @@
+﻿using DotSpatial.Projections;
+using ILICheck.Web.XtfLog;
+using NetTopologySuite.Features;
+using NetTopologySuite.Geometries;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ILICheck.Web
+{
+    /// <summary>
+    /// Provides helper methods for the GeoJSON (RFC 7946) format.
+    /// </summary>
+    public static class GeoJsonHelper
+    {
+        private static readonly ProjectionInfo lv95 = ProjectionInfo.FromEpsgCode(2056);
+
+        /// <summary>
+        /// Converts XTF log entries to a GeoJSON feature collection.
+        /// </summary>
+        /// <param name="logResult">The XTF log entries.</param>
+        /// <returns>A feature collection containing the log entries.</returns>
+        public static FeatureCollection CreateFeatureCollection(IEnumerable<LogError> logResult)
+        {
+            if (!AllCoordinatesAreLv95(logResult))
+            {
+                return null;
+            }
+
+            var features = logResult
+                    .Where(log => log.Geometry?.Coord != null)
+                    .Select(log => new Feature(ProjectLv95ToWgs84(log.Geometry.Coord), new AttributesTable(new KeyValuePair<string, object>[]
+                    {
+                        new ("type", log.Type),
+                        new ("message", log.Message),
+                        new ("objTag", log.ObjTag),
+                        new ("dataSource", log.DataSource),
+                        new ("line", log.Line),
+                        new ("techDetails", log.TechDetails),
+                    })));
```

Comment:
REQ: Die Enumeration mit `AsEnumerable()` abschliessen, ansonsten wir diese weiter unten im `foreach`-Loop immer wieder neu enumieriert/ausgeführt.

Reply:
`AsEnumerable()` macht nur einen Cast zu `IEnumerable<T>` (siehe https://stackoverflow.com/questions/2013846/why-use-asenumerable-rather-than-casting-to-ienumerablet), hier ist es aber bereits ein `IEnumerable<T>` und hat somit keinen Einfluss.

Reply:
Dann wäre `ToList()` die Lösung, um eine mehrfache Iteration zu vermeiden. Da wir jedoch gleich weiter unten die `features` nur einmal aufrufen und die Methode verlassen, macht es hier keinen Unterschied. Du kannst selber entscheiden, ob es belassen wird oder nicht :-).

Reply:
Dann werde ich es mal dabei belassen. Falls nötig könnte man später über die Collection iterieren.